src/wl/sys/wl_cfg80211_hybrid.c

```diff
@@ -1198,7 +1225,12 @@ static s32 wl_cfg80211_get_tx_power(struct wiphy *wiphy, s32 *dbm)
 	return err;
 }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 38)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 1, 0)
+static int wl_cfg80211_config_default_key(struct wiphy *wiphy,
```

Comment:
Ideally we would respect the style and indentation of the original function, and just interpolate the extra parameter whenever possible.